sources/src/develocity/build-scan.ts

```diff
@@ -19,15 +19,19 @@ export async function setup(config: BuildScanConfig): Promise<void> {
     maybeExportVariableNotEmpty('DEVELOCITY_INJECTION_ENFORCE_URL', config.getDevelocityEnforceUrl())
     maybeExportVariableNotEmpty('DEVELOCITY_INJECTION_DEVELOCITY_PLUGIN_VERSION', config.getDevelocityPluginVersion())
     maybeExportVariableNotEmpty('DEVELOCITY_INJECTION_CCUD_PLUGIN_VERSION', config.getDevelocityCcudPluginVersion())
-    maybeExportVariableNotEmpty('DEVELOCITY_INJECTION__PLUGIN_REPOSITORY_URL', config.getGradlePluginRepositoryUrl())
-    maybeExportVariableNotEmpty(
-        'DEVELOCITY_INJECTION__PLUGIN_REPOSITORY_USERNAME',
-        config.getGradlePluginRepositoryUsername()
-    )
-    maybeExportVariableNotEmpty(
-        'DEVELOCITY_INJECTION__PLUGIN_REPOSITORY_PASSWORD',
-        config.getGradlePluginRepositoryPassword()
-    )
+
+    let prefixes : string[] = ['DEVELOCITY_INJECTION', 'GRADLE']
+    prefixes.forEach(prefix => {
+        maybeExportVariableNotEmpty(`${prefix}_PLUGIN_REPOSITORY_URL`, config.getGradlePluginRepositoryUrl())
+        maybeExportVariableNotEmpty(
+            `${prefix}_PLUGIN_REPOSITORY_USERNAME`,
+            config.getGradlePluginRepositoryUsername()
+        )
+        maybeExportVariableNotEmpty(
+            `${prefix}_PLUGIN_REPOSITORY_PASSWORD`,
+            config.getGradlePluginRepositoryPassword()
+        )
+    })
```

Comment:
Given that there are only 4 permutations, I would just live with the minor amount of duplication and remove the `forEach`.